File: POO JAVA/Aula11/src/aula11/Bolsista.java
package aula11;

public class Bolsista extends Aluno{
    
    //Atributos da Classe
    private float bolsa;

    //Metodos Acessores e Modificadores
    public float getBolsa() {
        return bolsa;
    }
    public void setBolsa(float bolsa) {
        this.bolsa = bolsa;
    }
    
    //Metodos da Classe
    public void renovarBolsa(){
        System.out.println("Renovando a bolsa do "+this.nome+" do curso "+this.getCurso());
    }
    @Override
    public void pagarMensalidade(){
        System.out.println(this.nome+" e bolsista! Pagamento facilitado.");
    }
}

File: POO PHP/Aula11/Bolsista.php
<?php

require_once 'Aluno.php';

class Bolsista extends Aluno{

    //Atributos da Classe
    private $bolsa;
    
    //Metodo Acessor e Modificador
    public function getBolsa() {
        return $this->bolsa;
    }
    public function setBolsa($bolsa): void {
        $this->bolsa = $bolsa;
    }

    //Metodos da Classe
    public function renovarBolsa() {
        echo "Renovando bolsa do ".$this->getNome();
    }
    
    public function pagarMensalidade() {
        echo "Pagamento facilitado para bolsista.";
    }
}
